exalearn-ui/src/app/components/leave-test-modal/leave-test-modal.component.scss

```diff
@@ -9,20 +9,21 @@
 	width: 100%;
 }
 
+.mat-dialog-content {
+	overflow: visible;
+	display: flex;
+	flex-direction: row !important;
+	justify-content: space-evenly !important;
```

Comment:
I can see that it was used without `!important` before.
Can we avoid using `!important`?

Reply:
Refactored, ready